Update onboarding document (#970)
ebi-ait/hca-ebi-wrangler-central: docs/ebi-wrangler-onboarding.md

| Meeting                              | Day       | Time   | Frequency             | Notes                                |
|--------------------------------------|-----------|--------|-----------------------|--------------------------------------|
| Operations planning/review           | Wednesday | 9.30am | Weekly                | |

> | Dev sprint kick off meeting          | Wednesday | 10.30am| Bi-weekly             | At start of EBI sprint, only dev wrangler needs to attend |

Reviewer: ```suggestion
| Dev sprint kick off meeting          | Wednesday | 10.30am| Bi-weekly             | At start of EBI sprint, only wrangler on development needs to attend |
```

| EBI-DCP Sprint Planning & Demo       | Tuesday   | 2pm    | Bi-weekly             | 1 hour, telecon, at end of a sprint  |
| EBI DCP Sprint Retro                 | Tuesday   | 4pm    | Bi-weekly             | 1 hour, telecon                      |

> | User stories for next sprint         | Monday    | 10am   | Bi-weekly             | 1 hour, monday before end of EBI sprint |

Reviewer: ```suggestion
```
Don't think this is happening anymore

Reviewer 2: It is happening between me and Amnon (for now). No need to have it in this doc, certainly.

[#hca-development](https://embl-ebi-ait.slack.com/archives/C016R78CDV1) - channel for discussion about the current development work, sorting testing, answering questions etc

#hca-wrangler-metadata - this is where most of the discussion relating to wrangling and the HCA metadata are discussed

[#dcp-2](https://embl-ebi-ait.slack.com/archives/C01360XN04S) - This is a shared channel with the HCA slack. It is used for general announcements about the DCP or questions about the DCP, this is a good place to introduce yourself when you have logged in. 

> 

Reviewer: ```suggestion

[#dcp-ops](https://embl-ebi-ait.slack.com/archives/C9XD6L0AD) - This is a shared channel with the HCA slack. It is used for operation discussion/announcements, including when releases are available for review or they go live.

```

### Miscellanea

> - Printers: If you install the correspondent apps (You can search for them as “Printer”, under the “Managed software centre” app), you will be able to use the Konica printers in both buildings. Using them to print personal photocopies has a little cost (1 pence) that is automatically subtracted from your salary.

Reviewer 3: ```suggestion
- Printers: If you install the correspondent apps (You can search for them as “Printer”, under the “Managed software centre” app), you will be able to use the Konica printers in both buildings. Using the printers has a small cost which comes out of the group's budget.  
```
